pkg/deploy/deployment_keycloak.go

```diff
@@ -35,10 +35,10 @@ func NewKeycloakDeployment(cr *orgv1.CheCluster, keycloakPostgresPassword string
 
 	// add various certificates to Java trust store so that Keycloak can connect to OpenShift API
 	// certificate that OpenShift router uses (for 4.0 only)
-	addRouterCrt := "if [ ! -z \"${CHE_SELF__SIGNED__CERT}\" ]; then echo \"${CHE_SELF__SIGNED__CERT}\" > " + jbossDir + "/openshift.crt && " +
+	addRouterCrt := "if [ ! -z \"${CHE_SELF__SIGNED__CERT}\" ]; then echo \"${CHE_SELF__SIGNED__CERT}\" > " + jbossDir + "/che.crt && " +
 		"keytool -importcert -alias ROUTERCRT" +
 		" -keystore " + jbossDir + "/openshift.jks" +
-		" -file " + jbossDir + "/openshift.crt -storepass " + trustpass + " -noprompt; fi"
+		" -file " + jbossDir + "/che.crt -storepass " + trustpass + " -noprompt; fi"
```

Comment:
why the filename change?

Reply:
That makes it easier to debug certificate problems inside the Keycloak container if each certificate that has been added to the Keycloak truststore is still available on the disk.